.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release-plz
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - bindings/rust-primitives/**
+      - bindings/rust-bindings/**
+
+jobs:
+  release-plz:
+    name: Release-plz
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Commit Cargo.lock
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add Cargo.lock
```

Comment:
We have two `Cargo.lock` files, once under `/bindings/rust-primitives` and one under `/bindings/rust-bindings`.  So I don't think this command will work. 

I'm not sure exactly what other changes you need to make to take account of the two crates. 

Also, I don't see this step in the release-plz instructions https://release-plz.ieni.dev/docs/github/quickstart, curious about why it is needed?

Reply:
Oh good catch!

Reply:
I think it would be ok to leave the steps in if you want, the explanatory comment is good though. 

Can we modify the step to reference the actual `cargo.lock` files? Otherwise, if we activate the step at some later point it will fail and need fixing?